src/pages/EditReportFieldPage.tsx

```diff
@@ -105,10 +103,10 @@ function EditReportFieldPage({route, policy, report, policyReportFields}: EditRe
         return (
             <EditReportFieldDropdownPage
                 policyID={report.policyID ?? ''}
-                fieldID={reportField.fieldID}
+                fieldKey={fieldKey}
                 fieldName={Str.UCFirst(reportField.name)}
                 fieldValue={fieldValue}
-                fieldOptions={reportField.values}
+                fieldOptions={reportField.values.filter((value) => !(value in reportField.disabledOptions))}
```

Comment:
This was not the right way to check if a value is in an array in javascript. It caused this deploy blocker: https://github.com/Expensify/App/issues/38898

Reply:
Ugh... I hate when this happens 😢. I was super sure that this won't happen again.